app.js

```diff
@@ -14,22 +14,23 @@ app.get('/api', getData);
 app.get('/api/articles/:article_id', getArticleById);
 app.get('/api/articles', fetchArticles);
 
-
+app.all('/*', (req, res, next) => {
```

Comment:
You've added this middleware with no associated test on this commit? Please remove as we haven't been driven by our tests to write this code.